conans/client/tools/win.py

```diff
@@ -409,11 +409,11 @@ def vcvars_command(settings, arch=None, compiler_version=None, force=False, vcva
 
 
 def vcvars_dict(settings, arch=None, compiler_version=None, force=False, filter_known_paths=False,
-                vcvars_ver=None, winsdk_version=None, only_diff=True):
+                vcvars_ver=None, winsdk_version=None, only_diff=True, output=None):
```

Comment:
This `output` is new. Document it.

Reply:
No, this is only the "private" interface of this tool. So I added this parameter to avoid the warning. But you are right and I have to provide the "currified" version for the user